webgpu.c

```diff
@@ -604,10 +604,47 @@ void wgpuComputePassEncoderEnd(WGPUComputePassEncoder computePassEncoder)
 // {
 // }
 
-// void wgpuComputePassEncoderSetBindGroup(WGPUComputePassEncoder computePassEncoder, uint32_t groupIndex,
-//     WGPUBindGroup group, size_t dynamicOffsetCount, uint32_t const* dynamicOffsets)
-// {
-// }
+void wgpuComputePassEncoderSetBindGroup(WGPUComputePassEncoder computePassEncoder, uint32_t groupIndex,
+    WGPUBindGroup group, size_t dynamicOffsetCount, uint32_t const* dynamicOffsets)
+{
+    if(!computePassEncoder) unreachable();
+
+    wasi_webgpu_webgpu_set_bind_group_error_t err;
+
+    wasi_webgpu_webgpu_borrow_gpu_bind_group_t *group_wasi = NULL;
+    if (group) {
+        wasi_webgpu_webgpu_borrow_gpu_bind_group_t group_borrow = wasi_webgpu_webgpu_borrow_gpu_bind_group(group->bind_group);
+        group_wasi = &group_borrow;
```

Comment:
group_borrow goes out of scope here

Reply:
[This is Use-After-Scope (Dangling Pointers)](https://en.wikipedia.org/wiki/Dangling_pointer)